server/conn.go

```diff
@@ -133,6 +135,11 @@ func (cc *clientConn) Close() error {
 func (cc *clientConn) writeInitialHandshake() error {
 	data := make([]byte, 4, 128)
 
+	var serverCapability = defaultCapability
```

Comment:
if this global variable `defaultCapability` is not used anywhere else, you can just do it.